persone.rb

```diff
@@ -26,4 +29,8 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(book, date)
+    Rental.new(date, book, self)
```

Comment:
- [ ] Kindly use the arrangement of the Rental constructor parameters (book, person, _date).

  ```
  Rental.new(book, self, date)
  ```